back/app/services/project_copy_service.rb

```diff
@@ -342,10 +342,10 @@ def yml_phase_files(shift_timestamps: 0)
   end
 
   def yml_poll_questions(shift_timestamps: 0)
-    participation_context_ids = [@project.id] + @project.phases.ids
-    Polls::Question.where(participation_context_id: participation_context_ids).map do |q|
+    phase_ids = @project.phases.ids
+    Polls::Question.where(participation_context_id: phase_ids).map do |q|
```

Comment:
I think we could write `where(participation_context: Phase.where(project: @project))` and remove the line above to avoid running an SQL query by `@project.phases.ids`.